WebHostLib/templates/multitracker__Celeste64.html

```diff
@@ -0,0 +1,91 @@
+{% extends "multitracker.html" %}
+{% block head %}
+    {{ super() }}
+    <script type="application/ecmascript" src="{{ url_for("static", filename="assets/jquery.scrollsync.js") }}"></script>
+    <script type="application/ecmascript" src="{{ url_for("static", filename="assets/lttpMultiTracker.js") }}"></script>
+{% endblock %}
+
+{# List all tracker-relevant icons. Format: (Name, Image URL) #}
+{% set icons = {
+    "Strawberry"            : "https://github.com/PoryGone/Celeste-64-AP-Tracker/blob/main/images/items/strawberry.png?raw=true",
```

Comment:
While there's technically nothing wrong with it, I would like the images to be imported from a proper CDN, if possible. Is there an alternative place to host these images?

If the images ever change or are removed on the master branch of Pory's GitHub, the tracker layout will break.

Reply:
Put into my ap-icons repository (https://github.com/mrkssr/ap-icons) which is already used for the Undertale tracker. Pory is credited in tracker code and also mentioned in a text file in the repo.